ding/ding.py

```diff
@@ -12,7 +12,7 @@
 import argparse
 
 
-VERSION = '2.0.0'
+VERSION = '2.0.1'
```

Comment:
I think it's best if we leave the version as it is for now -- I typically do the versioning process in a release branch, and this would change would typically lead to a minor version bump i.e. 2.1.0 because it added a feature rather than fixed a bug (according to [semvar](http://semver.org/))